static/js/lib/ckeditor/plugins/constants.ts

```diff
@@ -64,3 +64,25 @@ export const TABLE_ALLOWED_ATTRS: string[] = ["colspan", "rowspan"]
  * with a double quote and captures anything in between the quotes.
  */
 export const ATTRIBUTE_REGEX = /(\S+)=["']?((?:.(?!["']?\s+(?:\S+)=|\s*\/?[>"']))+.)["']?/g
+
+export const LEGACY_SHORTCODES = [
```

Comment:
ok cool, I was too optimistic 😃

Reply:
> also am I right in thinking that there's no harm in having it in the list even after the issue has been address via your PR?

yes, i agree.

Reply:
If it's not there then the rule won't apply right?  There shouldn't be any harm to simply including it in this list, as even if we have a need for it in the future for some reason it would be manually inserted by us and won't be touched by editors.